index.html

```diff
@@ -25,13 +25,13 @@
     <div class="intro">
       <h1>The Portfolio of Carlo Santos</h1>
     </div>
-    <div class="about">
+    <div id="profile" class="about">
       <h1>About Me:</h1>
       <p>
         Ang Lorem Ipsum ay ginagamit na modelo ng industriya ng pagpriprint at pagtytypeset. Ang Lorem Ipsum ang naging regular na modelo simula pa noong 1500s, noong may isang di kilalang manlilimbag and kumuha ng galley ng type at ginulo ang pagkaka-ayos nito upang makagawa ng libro ng mga type specimen. Nalagpasan nito hindi lang limang siglo, kundi nalagpasan din nito ang paglaganap ng electronic typesetting at nanatiling parehas. Sumikat ito noong 1960s kasabay ng pag labas ng Letraset sheets na mayroong mga talata ng Lorem Ipsum, at kamakailan lang sa mga desktop publishing software tulad ng Aldus Pagemaker ginamit ang mga bersyon ng Lorem Ipsum.
       </p>
     </div>
-    <div class="projects">
+    <div class="projectsTemp">
```

Comment:
Make HTML attribute names not camel-case, i.e. Line 34 would be:

```
<div class="projects-temp">
```

Doing camel-case doesn't break things, but it's professional convention to not camel-case in HTML. Confusing, I know, but just a professional tip.